andrewinsoul/RoleManagementSystem
# set up webpack for developing with React

#### What does this PR do?
This PR setsup webpack for bundling the client assets

#### Description of Task to be completed?
- Install dependencies
- Configure webpack

#### How should this be manually tested?
- Fetch the branch locally 
- cd into client folder
- run npm install to install all dependencies
- run npm start
- webpack should build the assets and open a tab in your browser
